website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -28,24 +28,35 @@ We recommend using a combination of client-side and server-side validation for t
 
 Server-side validation provides a more thorough check on the server once the data has been submitted and helps keep our applications safe.
 
-![Example of server side form validation](/assets/components/form/primitives/form-primitives-validation-server.png =600x*)
+![Example of server side form validation](/assets/components/form/primitives/form-primitives-validation-server.png =600x\*)
```

Comment:
Think there's a typo here as this isn't rendering an image.

Reply:
Fixed this, thanks for pointing it out!